prereise/gather/solardata/nsrdb/sam.py

```diff
@@ -1,7 +1,12 @@
+import csv
+import os
+import tempfile
+
 import numpy as np
 import pandas as pd
-import PySAM.Pvwattsv7 as PVWatts
+import PySAM.Pvwattsv8 as PVWatts
```

Comment:
Should L#38 (in `calculate_power_pv`) still be using `"pvwattsv7"`?

Reply:
Updated.